examples/samples/README.md

```diff
@@ -0,0 +1,41 @@
+## Getting started with samples ##
```

Comment:
That's because I created this branch after those commits, I don't think deleting these commits would be a good idea, since that would delete them in the other PR as well. Merging them would be easier because I didn't make any changes to those files in this branch(this PR).